packages/flutter_tools/lib/src/build_system/targets/localizations.dart

```diff
@@ -0,0 +1,285 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+import 'package:process/process.dart';
+import 'package:yaml/yaml.dart';
+
+import '../../artifacts.dart';
+import '../../base/file_system.dart';
+import '../../base/io.dart';
+import '../../base/logger.dart';
+import '../../globals.dart' as globals;
+import '../build_system.dart';
+import '../depfile.dart';
+
+/// Run the localizations generation script with the configuration [options].
+Future<Depfile> generateLocalizations({
+  @required LocalizationOptions options,
+  @required String flutterRoot,
+  @required FileSystem fileSystem,
+  @required ProcessManager processManager,
+  @required Logger logger,
+  @required Directory projectDir,
+  @required String dartBinaryPath,
+}) async {
+  // Setup project inputs and outputs. This currently makes some
+  // guess about the output and does not include all files.
```

Comment:
Should we file/is there an issue about making this more precise?

Reply:
https://github.com/flutter/flutter/issues/55503

Reply:
Please add a TODO or reference the issue in a comment.